src/styles.css

```diff
@@ -31,4 +31,8 @@ body {
     display: grid;
     grid-template-columns: 3% auto 3%;
     grid-template-rows: 100%;
+}
+
+.toolbar-readonly {
```

Comment:
this can now be removed now that we are conditionally rendering the toolbar

Reply:
@DianaLease It's done, remove the unused CSS